tests/sentry/api/test_event_search.py

```diff
@@ -1948,6 +1948,15 @@ def test_project_name(self):
             in str(err)
         )
 
+    def test_not_has_project(self):
+        with pytest.raises(InvalidSearchQuery) as err:
```

Comment:
not sure why we don't use assertRaises but looks like the other tests in this class use pytest.raises too so it's probably better to be consistent